aviary/docs/user_guide/hamilton_standard.md

```diff
@@ -97,7 +97,7 @@ The newly added aviary options and variables are:
 Aircraft.Engine.PROPELLER_DIAMETER
```

Comment:
I'm seeing a bunch of stuff with PROPELLER as the first part of the name, I think it may be time to make this `.PROPELLER.` i.e. Aircraft.Engine.Propeller.DIAMETER

Reply:
I also think it makes sense to separate out the propeller analysis, and making it a subsection of engine probably makes the most sense (at least in the short term). Adding a new `Aircraft.Propeller` section will add extra work to the multi-engine effort

Reply:
It looks like there are actually only 8 names that fit in this category:

Aircraft.Engine.PROPELLER_INTEGRATED_LIFT_COEFFICIENT
Aircraft.Engine.PROPELLER_ACTIVITY_FACTOR
Aircraft.Engine.NUM_PROPELLER_BLADES
Aircraft.Engine.PROPELLER_DIAMETER
Aircraft.Design.MAX_PROPELLER_TIP_SPEED
Dynamic.Mission.PROPELLER_TIP_SPEED
Aircraft.Engine.HAS_PROPELLERS
Dynamic.Mission.INSTALLATION_LOSS_FACTOR